pkg/flow/adapter/jqtransformation/adapter.go

```diff
@@ -63,7 +66,9 @@ func NewAdapter(ctx context.Context, envAcc pkgadapter.EnvConfigAccessor, ceClie
 		replier:  replier,
 		ceClient: ceClient,
 		logger:   logger,
-		mt:       mt,
+
+		mt: mt,
```

Comment:
here and in the rest of flow group components: why do we need metric tag `mt` in the adapter object?